src/app/components/SideBar/SideBarItem/SideBarItem.tsx

```diff
@@ -20,19 +19,20 @@ export const SideBarItem = withRouter(
 				data-testid={`side-menu__item--${itemKey}`}
 			>
 				<li
-					className={`rounded-lg py-5 pl-10 pr-8 h-15 flex items-center ${
-						isActive ? "text-theme-primary-600 bg-theme-primary-100 dark:bg-theme-secondary-800" : ""
-					}`}
+					className={cn("rounded py-5 pl-10 pr-8 h-15 flex items-center", {
+						"text-theme-primary-600 dark:text-theme-secondary-200 bg-theme-primary-100 dark:bg-theme-secondary-800": isActive,
+					})}
 				>
 					{isActive && <div className="absolute top-0 bottom-0 left-0 w-2 rounded bg-theme-primary-600" />}
 
 					{icon && (
 						<div
-							className={`mr-3 ${
+							className={cn(
+								"mr-3",
 								isActive
-									? "text-theme-primary-600"
-									: "text-theme-primary-300 dark:text-theme-secondary-600"
-							}`}
+									? "text-theme-primary-600 dark:text-theme-secondary-200"
+									: "text-theme-primary-300 dark:text-theme-secondary-600",
```

Comment:
```suggestion
									{'text-theme-primary-300 dark:text-theme-secondary-600': ! isActive}
```
it can get styles from the parent

Reply:
Unfortunately this doesn't work since the color of the label and the color of the text aren't the same - it has to be set explicitly.

Reply:
actually when item is active the style of icon and label is the same [here](https://github.com/ArkEcosystem/desktop-wallet/pull/3705/files#diff-707759832c23746e0972cfc35acc3d2d070e0cda0ffd503f5759a40651dae1fcR23) and [here](https://github.com/ArkEcosystem/desktop-wallet/pull/3705/files#diff-707759832c23746e0972cfc35acc3d2d070e0cda0ffd503f5759a40651dae1fcR33)
and of course i checked my code
but no big deal

Reply:
Apologies, I stand corrected 👍🏻  I misread one of those numbers apparently.

Reply:
it's late (at least here) and this is normal
no need to apologize 😉